mainpage.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <meta charset="utf-8">
+    <title>Salmon Cookie Shop</title>
+  </head>
+  <nav>
+    <ul id="navigation">
+      <li><a href='file:///Users/codyhulsey/projects/cookie_stand_201/mainpage.html'>HOME</a></li>
+      <li><a href="file:///Users/codyhulsey/projects/cookie_stand_201/sales.html">SALES</a></li>
+      <li>
+      <a href="file:///Users/codyhulsey/projects/cookie_stand_201/styleguide.html">STYLE GUIDE</a></li>
+    </ul>
+  </nav>
+  <body>
+    <section class="mainPageBlock">
```

Comment:
Much `<section>`s. There are probably more semantically meaningful tags for you to use rather than these sections. Go check out html5doctor.com